importer/importer_utils.py

```diff
@@ -11,7 +11,21 @@
 
 site_cache = {}
 
-DATA_DIRECTORY = "data"
+
```

Comment:
Probably made this way more complicated than necessary, but I've substituted the references to DATA_DIRECTORY in these files with a function in `utils`.